tests/copilot_helper/conftest.py

```diff
@@ -59,7 +68,11 @@ def fakefs(fs):
 
 @pytest.fixture(scope="function", autouse=True)
 def mock_application():
-    with patch("dbt_copilot_helper.utils.application.load_application") as app_patch:
+    with patch(
+        "dbt_copilot_helper.utils.application.load_application",
+    ) as app_patch:
+        os.environ.pop("AWS_PROFILE", None)
```

Comment:
This is where the environment variable is unset. The rest are pre-commit hook changes...